cave/com.raytheon.viz.pointdata/localization/menus/obs/baseSurface.xml

```diff
@@ -8,9 +8,55 @@
 	<contribute xsi:type="bundleItem" file="bundles/SurfacePlotWinds.xml"
 			menuText="Surface Winds" id="MetarPlot">
     </contribute>
-	<contribute xsi:type="bundleItem" file="bundles/SurfacePlotMetar.xml"
+    <contribute xsi:type="bundleItem" file="bundles/SurfacePlotMetar.xml"
 			menuText="METAR Station Plot" id="MetarPlot">
     </contribute>
+    <contribute xsi:type="bundleItem" file="bundles/LDADMesoPlot.xml"
+        menuText="60 min RAWS Plot" id="local60minstnplot">
+        <substitute key="svg" value="ldadMesoDesign.svg" />
+        <substitute key="legend" value="60 min RAWS Data Plot" />
+        <substitute key="posOffset" value="1800" />
+        <substitute key="negOffset" value="1800" />
+    </contribute>
+    <contribute xsi:type="subMenu" menuText="Other RAWS Plots">
+        <contribute xsi:type="bundleItem" file="bundles/LDADMesoPlot.xml"
+        menuText="30 min RAWS Plot" id="local30minstnplot">
+        <substitute key="svg" value="ldadMesoDesign.svg" />
+        <substitute key="legend" value="30 min RAWS Data Plot" />
```

Comment:
Do we think all the submenu options are important? Since, like Pete is having, you'll end up with "missing" data from station that haven't reported in that time frame?

Reply:
I think it's still important to have the submenu options for specific cases when you are wanting to have just the latest data (don't want to be looking at data that's almost an hour old).